packages/editor-ui/src/components/canvas/elements/edges/CanvasEdge.vue

```diff
@@ -93,6 +95,8 @@ const segments = computed(() => renderData.value.segments);
 
 const labelPosition = computed(() => renderData.value.labelPosition);
 
+const isConnectorStraight = computed(() => renderData.value.isConnectorStraight);
```

Comment:
nitpick/optional - it would be nice if this was above `edgeToolbarStyle` since it's used there.